designs/capi-resources-custom-namespace.md

```diff
@@ -0,0 +1,54 @@
+# CAPI Resources in Custom Namespace
+
+## Introduction
+
+**Problem:** Customers managing large scale EKS-A clusters (hundreds or thousands of workload clusters for a given management cluster) expect to have difficulty managing all these resources if they are all in the same namespace 
```

Comment:
curious if we've had a specific user with an experience that indicates this feature is needed and what their experience has been